onlinebanking-hbci4java/src/main/java/hbci4java/Hbci4JavaBanking.java

```diff
@@ -150,6 +151,7 @@ public LoadBookingsResponse loadBookings(BankApiUser bankApiUser, BankAccess ban
             HBCIExecStatus status = hbciHandler.execute();
             if (!status.isOK()) {
                 LOG.error("Status of SaldoReq+KUmsAll+DauerSEPAList batch job not OK " + status);
+                throw new AccessLockedAccountException();
```

Comment:
Erroneous hbci response is not necessarily a locked account.

Reply:
Suggestion: check for errors at KUmsAll job
eg:
bookingsJob.getJobResult().getJobStatus().hasErrors()

and throw more general Exception